README.md

```diff
@@ -74,7 +74,7 @@ Follow these steps to set up and run the GitHub App, and get the webhook server
      - `GITHUB_WEBHOOK_SECRET`: the 'Webhook secret' string you generated
      - `GITHUB_APP_ID`: App ID at the top of the app settings
      - `GITHUB_PRIVATE_KEY_PATH`: relative path to .pem file (e.g., 'private-key.pem')
-     - `GITHUB_APP_USER_NAME`: the 'name' of your GitHub app, followed by '[bot]'
+     - `GITHUB_APP_USER_NAME`: 'Aiderbot[bot]'
```

Comment:
@Aiderbot The GITHUB_APP_USER_NAME will depend on the 'app' name that the user chooses when they create the app, so it will not be Aiderbot in this case.